apps/events/models.py

```diff
@@ -885,85 +819,91 @@ def get_minimum_rule_offset_for_user(self, user: User):
+            # Offset is currently 1 day if you have marks, regardless of amount.
+            mark_offset = timedelta(days=1)
+            minimum_user_offset += mark_offset
+
+        return minimum_user_offset
+
+    def _check_marks(self, user: User):
         expiry_date = get_expiration_date(user)
         if expiry_date and expiry_date > timezone.now().date():
             # Offset is currently 1 day if you have marks, regardless of amount.
             mark_offset = timedelta(days=1)
             rule_offset = self.get_minimum_rule_offset_for_user(user)
-            postponed_registration_start = (
-                self.registration_start + rule_offset + mark_offset
-            )
+
+            previous_offset = self.registration_start + rule_offset
+            postponed_registration_start = previous_offset + mark_offset
+
+            logger.debug(self.registration_start)
+            logger.debug(postponed_registration_start)
+            logger.debug(rule_offset)
 
             before_expiry = self.registration_start.date() < expiry_date
 
             if postponed_registration_start > timezone.now() and before_expiry:
                 if (
-                    "offset" in response
-                    and response["offset"] < postponed_registration_start
-                    or "offset" not in response
+                    previous_offset < postponed_registration_start
+                    or previous_offset == self.registration_start
                 ):
-                    response["status"] = False
-                    response["status_code"] = 401
-                    response["message"] = _("Din påmelding er utsatt grunnet prikker.")
-                    response["offset"] = postponed_registration_start
-        return response
-
-    def _process_rulebundle_satisfaction_responses(self, responses):
-        # Put the smallest offset faaar into the future.
-        smallest_offset = timezone.now() + timedelta(days=365)
-        offset_response = {}
-        future_response = {}
-        errors = []
-
-        for response in responses:
-            if response["status"]:
-                return response
-            elif "offset" in response:
-                if response["offset"] < smallest_offset:
-                    smallest_offset = response["offset"]
-                    offset_response = response
-            elif response["status_code"] == 402:
-                future_response = response
-            else:
-                errors.append(response)
-
-        if future_response:
-            return future_response
-        if smallest_offset > timezone.now() and offset_response:
-            return offset_response
-        if errors:
-            return errors[0]
-
-    def rules_satisfied(self, user):
+                    raise RegisterException(
+                        status=RegisterStatus.BLOCKED_MARK,
+                        offset=postponed_registration_start,
```

Comment:
As far as I can see, `offset` is supposed to be a `timedelta`, but here `postponed_registration_start` I think is a `datetime`? Since `self.registration_start` is a `datetime`?
Which might have been an uncaught error from the previous change?